src/dotnet/CarbonAware.WebApi/README.md

```diff
@@ -77,4 +77,15 @@ EG
   "embodiedEmissionsValue": null,
   "functionalUnitValue": null
 }
-```
\ No newline at end of file
+```
+
+
+## Error Handling
+
+The WebAPI uses the [.Net controller filter pipeline](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/filters?view=aspnetcore-6.0) to ensure that all requests respond with a consistent JSON schema.
```

Comment:
I'm not sure it's if worth expanding slightly here that not only is it consistent within the repo or with dotnet generally.

Reply:
I would definitely want more expansion on what "consistent" means here.